plugins/autogen/setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="composio_autogen",
-    version="0.3.9-rc.3",
+    version="0.3.9rc4",
```

Comment:
Please revert the version bump, we don't want perform a release with every single change

Reply:
I am not sure, how to proceed with this, should I match all the versions with master? master contains `0.3.9` everywhere I guess.